cli/src/main/java/org/adamalang/cli/runtime/Output.java

```diff
@@ -0,0 +1,33 @@
+package org.adamalang.cli.runtime;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.adamalang.ErrorTable;
+import org.adamalang.cli.Util;
+import org.adamalang.common.ErrorCodeException;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class Output {
+    /** Represents the different types of outputs, STDOUT changes based on class **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc